manifests/candlepin.pp

```diff
@@ -105,8 +105,7 @@
       target => $keystore,
       owner  => 'tomcat',
       group  => $group,
-    }
-
+    } ->
```

Comment:
So might be cleaner to put at requires on the directives below that need the password file.